### ISSUE TITLE
[v7.22.0] Changes in EncryptCookies breaks assertCookie test

### ISSUE THREAD

**Issue by SamuelWei** [laravel/framework#33671]
- Laravel Version: 7.22.0
- PHP Version: 7.4.0
- Database Driver & Version: sqlite

### Description:

With the introduction of the cookies prefix the assertCookie test function is broken.

### Steps To Reproduce:
1. Create a test route to set a cookie via the middleware:
```
public function test()
{
        return response('test')->cookie('name', 'value');
}
```

2. Add to routes
3. Write a basic test
```
public function testCookies()
{
        $response = $this->get('test');
        $response->assertCookie('name','value');
}
```
4. See the failing test:
Cookie [name] was found, but value [791a5xxxxxxxxxx0662|value] does not match [value].
Failed asserting that two strings are equal.
Expected :'value'
Actual   :'791a5xxxxxxxxxx0662|value'

**Comment by GrahamCampbell** [comment]
Please upgrade to 7.22.2. :)

**Comment by SamuelWei** [comment]
@GrahamCampbell I'm on the latest version and can't get it working. Looking at the source the cookie is decrypted inside the test (not using the middleware). As in the middleware the prefix was added, I also needs to be added inside this cookie test.

I believe you would have to add something like this to the code in https://github.com/laravel/framework/blob/7.x/src/Illuminate/Testing/TestResponse.php#L304
 ```
$prefix = hash_hmac('sha1', $cookie->getName().'v2', app('encrypter')->getKey());

PHPUnit::assertEquals(
           $prefix.'|'.$value, $actual,
            "Cookie [{$cookieName}] was found, but value [{$actual}] does not match [{$value}]."
 );
```

**Comment by driesvints** [comment]
@SamuelWei are you sure that you're on 7.22.2? Can you run `php artisan --version`?

**Comment by SamuelWei** [comment]
@driesvints Yes, I can confirm. Can you run my test without any problems?
![image](https://user-images.githubusercontent.com/4281791/88576209-5b38ee80-d045-11ea-8ab2-746c241893ad.png)
![image](https://user-images.githubusercontent.com/4281791/88576266-74419f80-d045-11ea-8392-ddcad3bc7189.png)

**Comment by driesvints** [comment]
@SamuelWei thanks. I think I see the problem. Looking into it now.

**Comment by SamuelWei** [comment]
Fixed in v7.22.4
Thanks for your quick fix @taylorotwell
